### DISCUSSION TITLE
Companies/organizations using Markdoc?

### DISCUSSION THREAD

**Discussion by mfix-stripe** [markdoc/markdoc#203] [Announcements]
Hey Markdoctors! Are you using Markdoc at your company or organization? I'd love to find out more about how you are using it, as well as what troubles you are having.

Additionally, if you are currently using Markdoc, we are interested in adding a few logos to our website (https://markdoc.dev). Let me know if this interested you by commenting below or shooting me an email at mfix@stripe.com.

**Comment by Patriksafar** [comment]
Hey, 👋 

we have started using Markdoc in [Twisto](https://www.twisto.cz/) for our public docs for merchants:
- https://docs.twisto.cz/
- https://docs.twisto.pl/ 

We are using Markdoc along with NextJs. It was really easy to use and I personally enjoyed using it. We are using [gray-matter](https://www.npmjs.com/package/gray-matter) for parsing files and creating simple search index for our internal search API. I would like to know how would you solve it differently. Is there any "built-in" possibility to do search functionality?

**Comment by samuelstroschein** [comment]
https://inlang.com/

We are using markdoc for the documentation site and (likely; soon) for marketplace items with a custom SolidJS renderer. The de-coupling of code (components) and authoring is the main appeal of markdoc for us.

**Reply by mfix-stripe** [reply]
This is cool! If you are up for it, please share the SolidJS renderer in our ["Show and Tell" forum](https://github.com/markdoc/markdoc/discussions/categories/show-and-tell) 🙂 

Thanks for using Markdoc!

**Comment by TorstenDittmann** [comment]
Hey 👋🏻 

we have launched our new website over at @appwrite using Markdoc for our documentation, blog, tutorials & co.

Website: https://appwrite.io
Source: https://github.com/appwrite/website

On that journey I've also wrote [TorstenDittmann/svelte-markdoc-preprocess](https://github.com/TorstenDittmann/svelte-markdoc-preprocess) in order to use Markdoc to its full extent with Svelte Kit.
